include/xen/interface/hvm/hvm_op.h

```diff
@@ -1,4 +1,6 @@
 /*
+ * SPDX-License-Identifier: GPL-2.0
```

Comment:
Here and elsewhere: if you put SPDX license then any explicit license text must be removed. E.g. the below is already covered by the SPDX identifier and must be removed.